ui/apps/platform/src/Components/CVEStackedPill/CVEStackedPill.js

```diff
@@ -54,12 +56,23 @@ const CVEStackedPill = ({
 
     const width = horizontal ? '' : 'min-w-16';
 
-    const imageScanMessages = getImageScanMessages(imageNotes || [], scan?.notes || []);
-    const hasScanMessages = Object.keys(imageScanMessages).length > 0;
+    let scanMessagesNonConst = {};
+    let entityNonConst = 'Entity';
+    if (imageNotes?.length > 0) {
+        scanMessagesNonConst = getImageScanMessages(imageNotes || [], scan?.notes || []);
+        entityNonConst = 'Image';
+    } else if (nodeNotes?.length > 0) {
+        scanMessagesNonConst = getNodeScanMessages(nodeNotes || [], scan?.notes || []);
+        entityNonConst = 'Node';
+    }
+
+    const scanMessages = scanMessagesNonConst;
+    const hasScanMessages = Object.keys(scanMessages).length > 0;
+    const entity = entityNonConst;
```

Comment:
Since we now have two possible types of notes, we should pull the logic up and out to the "caller", and just display them if they are passed in.

nit: `entityNonConst` and `scanMessagesNonConst` sound like Golang naming conventions